gdrive/struct.go

```diff
@@ -0,0 +1,21 @@
+package gdrive
+
+import "net/http"
+
+type GoogleDriveStorage struct {
```

Comment:
Please copy this struct from other providers' packages and edit as needed so the documentation comments are maintained.